DiffMatchPatch.zig

```diff
@@ -69,7 +72,7 @@ patch_delete_threshold: f32 = 0.5,
 /// Chunk size for context length.
 patch_margin: u16 = 4,
 
-pub const DiffError = error{OutOfMemory};
+pub const DiffError = error{ OutOfMemory, SystemClockUnavailable };
```

Comment:
Diffing shouldn't fail just because the system time is unavailable. Instead, timeouts should not be performed.